opennms-base-assembly/src/main/resources/contrib/opennms.mib

```diff
@@ -95,6 +98,18 @@ syslogMessageObjects	OBJECT IDENTIFIER
 correlatedEventObjects	OBJECT IDENTIFIER
 							::= { eventsSpecific 3 }
 
+opennmsProducts		OBJECT IDENTIFIER
+							::= { openNMS 42 }
+
+appliance		OBJECT IDENTIFIER
+							::= { opennmsProducts 5 }
+
+virtualAppliance	OBJECT IDENTIFIER
+							::= { appliance 1 }
+
+applianceMini		OBJECT IDENTIFIER
+							::= { appliance 2 }
+
```

Comment:
LGTM overall.  Can we add a `appliance1U` entry as well?

For different "versions" of the same appliance class, suppose we'll use a convention like "applianceMiniV2" ?

Reply:
Just pushed a new commit with appliance1U included.